assistant_dists/dream_ranking_and_sfc_based_dm/docker-compose.override.yml

```diff
@@ -0,0 +1,444 @@
+services:
+  agent:
+    command: sh -c 'bin/wait && python -m deeppavlov_agent.run agent.channel=telegram agent.telegram_token=$TG_TOKEN agent.pipeline_config=assistant_dists/dream_Speech_models_based/pipeline_conf.json agent.db_config=assistant_dists/dream_Speech_models_based/db_conf.json'
```

Comment:
dist name

Reply:
checked, the name is correct now, `dream_ranking_and_sf_based_dm`